### PR TITLE
PROC-1357 Create v2 profiles/me api endpoint

### PR DESCRIPTION
**Pull request by MollieRick**

### CODE REVIEW

File: source/reference/v2/profiles-api/get-profile-me.rst [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+Retrieve the profile that is authenticated with the current API Key.
+
+Parameters
+----------
+No parameters applicable for this endpoint.
+
+Response
+--------
+``200`` ``application/json; charset=utf-8``
```
Comment (willemstuursma) [inline source/reference/v2/profiles-api/get-profile-me.rst]: Just refer to the GET Profile response instead of copying it all again here.

**Comment by willemstuursma** [comment]
Can you update the changelog

**Comment by MollieRick** [comment]
@willemstuursma Updating the changelog; with date of monday?

File: source/reference/v2/profiles-api/get-profile-me.rst (hunk 1 above)
Comment (MollieRick) [reply source/reference/v2/profiles-api/get-profile-me.rst]: Get rid of the examples as well?

**Comment by willemstuursma** [comment]
Yeah, I guess you can make Monday!

File: source/reference/v2/profiles-api/get-profile-me.rst (hunk 1 above)
Comment (willemstuursma) [reply source/reference/v2/profiles-api/get-profile-me.rst]: What do you think @MollieRick

File: source/reference/v2/profiles-api/get-profile-me.rst [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+.. endpoint::
+   :method: GET
+   :url: https://api.mollie.com/v2/profiles/me
+
+.. authentication::
+   :api_keys: true
+   :oauth: false
+
+Retrieve the profile that is authenticated with the current API Key.
```
Comment (willemstuursma) [inline source/reference/v2/profiles-api/get-profile-me.rst]: "Use this API if you are creating a plugin or SaaS application that allows users to enter a Mollie API key, and you want to give a confirmation of the website profile that will be used in your plugin or application. "

File: source/changelog/v2/changelog.rst [hunk 3]
```diff
@@ -10,7 +10,7 @@ September 2018
 Monday, 10th
 ------------
 
-- Added a new endpoint for getting the Profile of the used API Key. For details, see:
+- Added a new endpoint for retrieving the Profile of the used API key.. For details, see:
```
Comment (shirshir) [inline source/changelog/v2/changelog.rst]: Double `.`

Comment (MollieRick) [reply source/changelog/v2/changelog.rst]: Whoops... :-)

**Comment by willemstuursma** [comment]
@MollieRick can you add some references to https://docs.mollie.com/reference/v2/organizations-api/current-organization to point to your new API and vice versa.

**Pull request merged**